### ISSUE TITLE
Smiley question broken w/o javascript

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#80]
text to name "other:" and smileys do not appear when javascript is turned off.

**Comment by sirblackheart** [comment]
Yes i can reproduce this. Not sure if we can do much about it, the js is generated by lime query.

**Comment by markus2330** [comment]
Yes, if would be hidden by JS, non-JS user would see all possibilities. If you see some way to change the behaviour, please do so.
